examples/pylint/w0130_duplicate_value.py

```diff
@@ -0,0 +1,2 @@
+incorrect_set = {'value 1', 2, 3, 'value 1'}
```

Comment:
Good example! Please put the comment `# Error on this line` after the first line (which we try to do on all checker examples).